scripts/bracket_orders.js

```diff
@@ -106,9 +60,10 @@ module.exports = async callback => {
     // check price against dex.ag's API
     const targetTokenId = argv.targetToken
     const stableTokenId = argv.stableToken
-    const priceIsOk = await isPriceReasonable(exchange, targetTokenId, stableTokenId, argv.targetPrice)
+    const priceCheck = await isPriceReasonable(exchange, targetTokenId, stableTokenId, argv.targetPrice, artifacts)
+    const proceed = await proceedAnyways(priceCheck)
 
-    if (priceIsOk) {
+    if (priceCheck || proceed) {
```

Comment:
I'd consider having `proceedAnyways` be a function with input just a warning message (like "Price check failed!"), so it can be reused for different warnings. Then `proceed` is not needed anymore and you can write something like:
```
if (priceCheck || await proceedAnyways("Price check failed!")) {
```

Reply:
Also no input at all would be an option, since we always log any issue that occurs.

Reply:
I like your solution. While mine is working in principle, having one less variable makes it less confusing for sure. Great